tests/test_database_types.py

```diff
@@ -150,9 +211,12 @@
 # target_type: (int, bigint) }
 for source_db, source_type_categories in DATABASE_TYPES.items():
     for target_db, target_type_categories in DATABASE_TYPES.items():
-        for type_category, source_types in source_type_categories.items():  # int, datetime, ..
+        for (
+            type_category,
+            source_types,
+        ) in source_type_categories.items():  # int, datetime, ..
             for source_type in source_types:
-                for target_type in target_type_categories[type_category]:
+                for target_type in target_type_categories.get(type_category, []):
```

Comment:
We'll never know if a typo caused some the tests to not run.

Reply:
Fair! Reason I had it was to just be able to comment out something quickly, but I see now that the risk isn't worth it. Agree